resources/language/English/strings.po

```diff
@@ -724,4 +724,12 @@ msgstr ""
 
 msgctxt "#32184"
 msgid "Attempt secondary show title search"
+msgstr ""
+
+msgctxt "#32185"
+msgid "Exclude Plugins"
+msgstr ""
+
+msgctxt "#32186"
+msgid "Allow script exclusions"
```

Comment:
to me "Exclude scripts" sounds like it disables all scripts.... but, whatever works for you.